web/app/Http/Controllers/PageController.php

```diff
@@ -28,45 +214,29 @@ public function show(): View
         $timestamp = null;
         $tags = null;
 
-        if ($values['site']) {
-            $title = $values['site']->getName();
+        $title = $site->getName();
+
+        // TODO: this doesn't seem elegant?
+        $sidebar_page = Page::findSlug($site_id, 'nav:side', false, true);
+        if ($sidebar_page !== null) {
+            $sidebar_content = $sidebar_page->compiled_html;
         }
 
-        if ($values['wikiPage']) {
-            $page = $values['wikiPage'];
+        $page_content = $page->compiled_html;
+        $tags = $page->tags;
+        $revision = $page->revision_number;
+        $timestamp = $page->lastUpdated()->getTimestamp();
+        $title = $page->title;
+        $alt_title = $page->alt_title;
+        $social_title = $title;
```

Comment:
"social"?

Reply:
Title that shows up on social media websites.